modules/external-api/src/main/java/org/opencastproject/external/endpoint/LifeCycleManagementEndpoint.java

```diff
@@ -0,0 +1,593 @@
+            @RestResponse(description = "The specified lifecycle policy instance does not exist.", responseCode = HttpServletResponse.SC_NOT_FOUND),
+            @RestResponse(description = "The user doesn't have the rights to make this request.", responseCode = HttpServletResponse.SC_FORBIDDEN),
+            @RestResponse(description = "The request is invalid or inconsistent.", responseCode = HttpServletResponse.SC_BAD_REQUEST),
+        })
+    public Response getPolicy(
+        @HeaderParam("Accept") String acceptHeader,
+        @PathParam("id") String id) {
+        try {
+            LifeCyclePolicy policy = service.getLifeCyclePolicyById(id);
+
+            return ApiResponses.Json.ok(acceptHeader, policyToJson(policy));
+        } catch (NotFoundException e) {
+            return ApiResponses.notFound("Cannot find playlist instance with id '%s'.", id);
+        } catch (UnauthorizedException e) {
+            return Response.status(Response.Status.FORBIDDEN).build();
+        } catch (IllegalStateException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+    }
+
+    @GET
+    @Path("policies")
+    @RestQuery(
+        name = "policies",
+        description = "Get policies. Policies that you do not have read access to will not show up.",
+        returnDescription = "A JSON object containing an array.",
+        restParameters = {
+            @RestParameter(name = "limit", isRequired = false, type = INTEGER,
+                description = "The maximum number of results to return for a single request.", defaultValue = "100"),
+            @RestParameter(name = "offset", isRequired = false, type = INTEGER,
+                description = "The index of the first result to return."),
+            @RestParameter(name = "sort", isRequired = false, type = STRING,
+                description = "Sort the results based upon a sorting criteria. A criteria is specified as a pair such as:"
+                    + "<Sort Name>:ASC or <Sort Name>:DESC. Adding the suffix ASC or DESC sets the order as ascending or"
+                    + "descending order and is mandatory. Sort Name is case sensitive. Supported Sort Names are 'title'"
+                , defaultValue = "title:ASC"),
+        },
+        responses = {
+            @RestResponse(description = "Returns the playlist.", responseCode = HttpServletResponse.SC_OK),
+            @RestResponse(description = "The request is invalid or inconsistent.", responseCode = HttpServletResponse.SC_BAD_REQUEST),
+        })
+    public Response getPoliciesAsJson(
+        @HeaderParam("Accept") String acceptHeader,
+        @QueryParam("limit") int limit,
+        @QueryParam("offset") int offset,
+        @QueryParam("sort") String sort) {
+        if (offset < 0) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        if (limit < 0) {
```

Comment:
Are we allowing limit=0 to mean unlimited results?  If not then

```suggestion
        if (limit < 1) {
```